libcontainer/nsenter/nsexec.c

```diff
@@ -115,13 +119,10 @@ struct nlconfig_t {
  * Use the raw syscall for versions of glibc which don't include a function for
  * it, namely (glibc 2.12).
  */
-#if __GLIBC__ == 2 && __GLIBC_MINOR__ < 14
-#  define _GNU_SOURCE
-#  include "syscall.h"
+#if GLIBC_VERSION < 2014	/* < 2.14 */
```

Comment:
Why not just keep `#if __GLIBC__ == 2 && __GLIBC_MINOR__ < 14`

Reply:
In the latest version I'm about to push, I switched them to using `__GLIBC_PREREQ` which is even simpler.

While nobody is going to be using glibc 1.x and glibc has said there will never be a 3.x, technically doing the version check this way is not correct.